### PR TITLE
Update elasticsearch.md

### PR DESCRIPTION
**Pull request by jonashrem**
Clarified Elasticsearch documentation

## Purpose of this pull request

As Elasticsearch has no relevant breaking changes between minor versions (see https://www.elastic.co/guide/en/beats/libbeat/current/breaking-changes.html ), Elasticsearch 7.x should generally be remanded instead of 7.6.

I also renamed the html anchor to match Elasticsearch version.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/install-gde/prereq/elasticsearch.html

### CODE REVIEW

**Comment by jeff-matthews** [comment]
@jonashrem, in this context, "supported" means that ES 7.6 is the only version of the ES 7.x release line that we have officially tested with Magento. We cannot claim support for all ES 7.x versions.

**Comment by jeff-matthews** [comment]
However, I suppose that the anchor updates are still valid, so I'll reopen this.

**Comment by jonashrem** [comment]
I see. I have reverted that part.

What is your opinion about a hint that only Elasticsearch 7.6 has been tested but other 7.x Versions "should work in general"?

**Comment by jeff-matthews** [comment]
That's probably a good idea @jonashrem. We have a hint like that on the [main system requirements page](https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html#elasticsearch), so it makes sense to include it here as well.

**Comment by jonashrem** [comment]
yes, there is also a formulations like this for database ( https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html#database )

**Comment by jonashrem** [comment]
I just added the information.

File: src/guides/v2.4/install-gde/prereq/elasticsearch.md [hunk 1]
```diff
@@ -23,7 +23,9 @@ As of Magento 2.4, all installations must be configured to use [Elasticsearch][]
 
 ## Supported versions {#es-spt-versions}
 
-You must install and configure Elasticsearch 7.6.x before upgrading to Magento 2.4.0.
+You must install and configure Elasticsearch 7.6.x before upgrading to Magento 2.4.x.
+
+Be aware, that Magento 2.4.x is only tested with Elasticsearch 7.6.x at the moment. You can use other (7.x) versions at your discretion, but we recommend using the tested version of Elasticsearch.
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/elasticsearch.md:28]:
```suggestion
Magento 2.4.x is tested with Elasticsearch 7.6.x only. You can use other 7.x versions at your discretion, but we recommend using the tested version of Elasticsearch.
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
Some minor editorial suggestions

**Review by jeff-matthews — APPROVED** [review]
Thanks @jonashrem!

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**